TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -135,24 +135,24 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
         case .account:
             self.pushAccountSelection()
         case .orderAction:
-            self.selectionViewController.title = "Select " + ticketRow.getTitle(forOrder: self.order)
+            self.selectionViewController.title = "Select " + ticketRow.getTitle(forOrder: self.order).lowercased()
```

Comment:
Maybe move this in to the `pushOrderCapabilitiesSelection` function so you don't have to repeat this code